src/yang/ietf-bgp-policy.yang

```diff
@@ -248,6 +253,92 @@ module ietf-bgp-policy {
           }
         }
       }
+
+      container prefix-sets {
+        description
+          "Definitions for a list of IPv4 or IPv6 prefixes which
+           are matched as part of a policy.";
+
+        list prefix-set {
+          key "name";
+          description
+            "List of the defined prefix sets";
+
+          leaf name {
+            type string;
+            description
+              "Name of the prefix set used to reference the set in
+               match conditions.";
+          }
+
+          leaf mode {
+            type enumeration {
```

Comment:
What does our extension model look like for updating this?
For example, while matching ipv4 and ipv6 prefixes is a core common application, what happens if you want your policy to match a vpnv4 prefix?  A rt-constrain prefix?  Etc.

Reply:
We will just have you augment it. :(

Reply:
Right.  See prior discussions about the issues with augmenting for enumerations.  You can end up with inconsistent enumerations as a side effect.

Mahesh had a pattern that moved us to identities for the community types.  That might be appropriate here?